code/script.js

```diff
@@ -30,18 +38,142 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
+// Question 1
 // Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+const askQuestion1 = () => {
+  showMessage(`Hello, what's your name?`, 'bot')
 }
+setTimeout(askQuestion1, 500)
 
-// Set up your eventlisteners here
+const handleQuestion1 = event => { //the handleQuestion is only used in event listeners ie if we want to do something when we click or submit
+  event.preventDefault() //prevents refresh on submit
+  const nameInput = document.getElementById('name-input')
+  username = nameInput.value //store input value 
+  showMessage(`My name is ${username}`, 'user')
+  nameInput.value = '' //clears name input to empty string
+  setTimeout(() => askQuestion2(), 1000)
+  // setTimeout(() => selectDestination(name), 1000)  //pass arguments to selectDestination function with 1s delay --- update: no need to pass argument since we store it in global var
+}
+
+// Question 2
+const askQuestion2 = () => {
+  showMessage(`Where do you want to be picked up from ${username} ?`, 'bot') //show next quesiton from bot
+  inputWrapper.innerHTML = //create a new form - the previous form is deleted ALSO added autofocus to the input so the cursor goes to the input and the user doesn't have to click to type an answer
+    ` 
+    <form id="address-form">
+      <input id="address-input" type="text" autofocus /> 
```

Comment:
Very good with autofocus, also great explanation in your comment. 
// Ida & Sherin